smoke-tests/designer/wdio.conf.js

```diff
@@ -1,6 +1,6 @@
 const { hooks } = require("./support/hooks");
 const drivers = {
-  chrome: { version: "98.0.4758.102" }, // https://chromedriver.chromium.org/
+  chrome: { version: "100.0.4896.60" }, // https://chromedriver.chromium.org/
```

Comment:
I've seen this change in [another PR](https://github.com/XGovFormBuilder/digital-form-builder/pull/829/files#diff-4335e7563ea8c424c5534b178892f25967932dcb68161560998a75ef993863c7R3)

Should there be a separate PR just to fix this specific issue?

Reply:
actually, yes, that would be useful ☝️ (it's to do with wdio/selenium.. I'm sure you've heard enough of my rants about it so I won't bore you again lol)

Reply:
Yeah iv been meaning to bring this up, is there no way of auto checking the latest version to avoid us all having to manually update? (question for the issue perhaps)